src/components/SummitSubNav.js

```diff
@@ -18,13 +19,14 @@ function SubNav(props) {
     <nav className="subnav-bar">
       <div className="container">
         <div className="location">
-          <a href="/summit" >OpenInfra Summit</a>
+          <LinkComponent href="/summit" >OpenInfra Summit</LinkComponent>
         </div>
 
         <ul id="links-list" className="links-list">
-          <li><a id="summit" href="/summit" className="link">Home</a></li>
-          <li><a id="summit-sponsor" href="/summit-sponsor" className="link">Sponsors</a></li>
-          <li><a id="summit-tracks" href="/summit-tracks" className="link">Tracks</a></li>
+          <li><LinkComponent id="summit" href="/summit" className="link">Home</LinkComponent></li>
+          <li><LinkComponent id="summit-sponsor" href="/summit-sponsor" className="link">Sponsors</LinkComponent></li>
+          <li><LinkComponent id="summit-tracks" href="/summit-tracks" className="link">Tracks</LinkComponent></li>
+          <li><LinkComponent id="summit-schedule" href="/summit/schedule" className="link">Schedule</LinkComponent></li>
```

Comment:
its here we need a drop down as is spect out on the ticket
this drop down should be only displayed when the user is logged in and will have the schedule and my schedule as sub items